src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let results = days * 40;
+
+  if (days >= 7) {
+    results = results - 50;
```

Comment:
Consider using a constant for the value `50` to improve code readability and maintainability.